### DISCUSSION TITLE
Unable to disable auto installation of selene

### DISCUSSION THREAD

**Discussion by rameshsanth** [jay-babu/mason-null-ls.nvim#100] [Q&A]
I'm using redhat8 based linux. Automatic installation of selene uses incompatible GLIBC unfortunately. 
However I have downloaded and installed selene in a different directory. But when neovim is launched, it keeps installing latest version from GitHub and complains of incompatibility. below are some screenshots seen.
Distro: Redha8
Nvim dist: astronvim v4

My mason-null-ls config. I've set exclude = { selene }
![Screenshot 2024-03-24 at 11 33 58 AM](https://github.com/jay-babu/mason-null-ls.nvim/assets/22250291/bc1a90e2-0842-4e54-8016-81739fa41b03)

Mason still installs the
![Screenshot 2024-03-24 at 11 34 52 AM](https://github.com/jay-babu/mason-null-ls.nvim/assets/22250291/92c05fdc-976b-4c3d-8dbb-e359e1240411)

Null-ls logs
![Screenshot 2024-03-24 at 11 36 29 AM](https://github.com/jay-babu/mason-null-ls.nvim/assets/22250291/b55ebdfe-5992-4629-a93a-a543972e2b83)

**Comment by rameshsanth** [comment]
because of this binary compatibility, this plugin is being installed every time I launch Nvim which is very annoying

**Reply by jay-babu** [reply]
i am a little confused on why it is in ensure installed?

**Reply by jay-babu** [reply]
can you give me the output of what you get here before and after this? https://github.com/jay-babu/mason-null-ls.nvim/blob/main/lua/mason-null-ls/automatic_installation.lua#L59

**Reply by jay-babu** [reply]
@nsirimo too

**Reply by nsirimo** [reply]
Forgive my noobness but I'm using this with AstroNvim. This is what my mason.lua looks like.

```---@type LazySpec
return {
  -- use mason-lspconfig to configure LSP installations
  {
    "williamboman/mason-lspconfig.nvim",
    -- overrides `require("mason-lspconfig").setup(...)`
    opts = function(_, opts)
      -- add more things to the ensure_installed table protecting against community packs modifying it
      opts.ensure_installed = require("astrocore").list_insert_unique(opts.ensure_installed, {
        "lua_ls",
        -- add more arguments for adding more language servers
      })
    end,
  },
  -- use mason-null-ls to configure Formatters/Linter installation for null-ls sources
  {
    "jay-babu/mason-null-ls.nvim",
    -- overrides `require("mason-null-ls").setup(...)`
    opts = function(_, opts)
      -- add more things to the ensure_installed table protecting against community packs modifying it
      opts.ensure_installed = require("astrocore").list_insert_unique(opts.ensure_installed, {
        "prettier",
        "stylua",
        -- add more arguments for adding more null-ls sources
      })
    end,
  },
  {
    "jay-babu/mason-nvim-dap.nvim",
    -- overrides `require("mason-nvim-dap").setup(...)`
    opts = function(_, opts)
      -- add more things to the ensure_installed table protecting against community packs modifying it
      opts.ensure_installed = require("astrocore").list_insert_unique(opts.ensure_installed, {
        "python",
        -- add more arguments for adding more debuggers
      })
    end,
  },
}
```

**Comment by nsirimo** [comment]
I'm having the same issue

**Reply by marcopiraccini** [reply]
Same for me on a RPI5
